### ISSUE TITLE
Npcs that need to be touched

### ISSUE THREAD

**Issue by Henrybk** [OpenKore/openkore#38]
Openkore has no good way of managing npcs that start conversation on their own when we get close to them, there should be a way to add this to both talk with them to skip unwanted npc chat (the sign and gael bolt quests) and to add suport for portals that interact this way (rachel airplane, nidhogg instance, rogue quest).

**Comment by windhamwong** [comment]
Yes, the current NPC chat during walk or pass by is poor supported. However, it should be easily fixed in servertype in the NPC chat function.

**Comment by moneyisgod** [comment]
I guess task talknpc is doing this ?

**Comment by Nipodemos** [comment]
with autoTalkCont now working normally, conversatitions that does not have an answer will end automatic.
However, if the npc makes your answer, you get stuck there anyways... so still have to do this

**Comment by Fadreus** [comment]
#2548
